website/package.json

```
@@ -0,0 +1,47 @@
+{
+  "name": "website",
```

[optional, no idea if this matters]
```suggestion
  "name": "EUI+ docs website",
```